NCrontab/CrontabSchedule.cs

```diff
@@ -100,6 +100,12 @@ public sealed partial class ParseOptions
 
         public static CrontabSchedule? TryParse(string expression) => TryParse(expression, null);
 
+        public static bool TryParse(string expression, out CrontabSchedule? schedule)
+            => (schedule = TryParse(expression, null))!=null;
```

Comment:
Formatting inconsistencies:

```suggestion
        public static bool TryParse(string expression, out CrontabSchedule? schedule) =>
            (schedule = TryParse(expression, null)) != null;
```